timemachine/fe/estimator.py

```diff
@@ -1,19 +1,45 @@
 import copy
 import dataclasses
+import time
 from collections import namedtuple
 from typing import List, Tuple
 
 import numpy as np
+import pymbar
 
+from timemachine.fe import endpoint_correction, standard_state
+from timemachine.fe.utils import extract_delta_Us_from_U_knk, sanitize_energies
 from timemachine.lib import custom_ops, potentials
+from timemachine.md import minimizer
 from timemachine.md.states import CoordsVelBox
 from timemachine.parallel.client import SerialClient
 
 
 @dataclasses.dataclass
 class SimulationResult:
     xs: np.array
-    du_dls: np.array
+    boxes: np.array
+    lambda_us: np.array
+
+
+FreeEnergyModel = namedtuple(
+    "FreeEnergyModel",
+    [
+        "unbound_potentials",
+        "endpoint_correct",
+        "client",
+        "box",
+        "x0",
+        "v0",
+        "integrator",
+        "barostat",
+        "lambda_schedule",
+        "equil_steps",
+        "prod_steps",
+        "beta",
+        "prefix",
+    ],
+)
```

Comment:
nit: how would you feel about using the style:

```python
class FreeEnergy(NamedTuple):
    unbound_potentials: List[Potential]
    # ...
```
(main benefit being that it allows typing the elements)

Reply:
Switched this to a data class in [8dae286](https://github.com/proteneer/timemachine/pull/700/commits/8dae286198c36e65ec9a0afac2ec4cfe5821a774)